src/zulqer/narcissisticNumbers/__tests__/index.test.ts

```diff
@@ -0,0 +1,29 @@
+import { getNarcissisticNumbersInRange, isNarcissisticNumber } from '..';
+
+describe('find if provided number is Narcissistic using isNarcissisticNumber function ', function () {
```

Comment:
Since, you are already using anonymous function, wouldn't it be nice if you replace that with arrow function?

Reply:
yes .